### DISCUSSION TITLE
Use Image component as a wrapper for next/image component?

### DISCUSSION THREAD

**Discussion by albertbronsky** [mantinedev/mantine#719] [Q&A] [answered]
I'd like to know is it possible to use Mantine's `Image` component (for styling purposes) together with [next/image](https://nextjs.org/docs/api-reference/next/image) component of Next.js (for built-in image optimization)?

**Comment by rtivital** [comment, marked as answer]
No, this option is not supported, you should use next/image instead of Mantine Image when you work with Next.js

**Reply by albertbronsky** [reply]
Thanks. Without opening an additional thread, I'd like to ask is it possible to use Mantine's `Avatar` component with `next/image`?

**Reply by rtivital** [reply]
Nope, same as with Image, if you want Next.js optimization features you would want to use next/image instead. Though usually, avatars are stored on CDN and do not require optimization.
